styles/styles.css

```diff
@@ -0,0 +1,51 @@
+nav {
+    background-color: steelblue;
+    padding: 10px;
+}
+
+nav a {
+    color: white;
+    display: block;
+    padding: 20px;
+}
+
+h1 {
+    color: steelblue;
+}
+
+main img {
+    width: 150px;
+    height: auto;
+    border: 1px solid steelblue;
+    padding: 2px;
+    box-shadow: 0 0 30px gray;
+}
+
+p {
+    padding: 0px;
+}
+
+main {
+    max-width: 840px;
+}
+
+aside {
+    width: 200px;
+    margin: 14px;
+    border: 3px green;
+    padding: 4px;
+    background-color: yellow;
+    color: blue;
+}
+
+aside img {
+    width: 200px;
+    height: auto;
+}
+
+footer {
+    display: block;
+    margin-top: 20px;
+    border-top: 4px dotted black;
+    text-align: center;
+}
```

Comment:
### Error Message
![image](https://github.com/user-attachments/assets/14be32b6-e459-4f4e-b8af-560c7ec5dae7)

### Solution
Refer back to the course content to resolve this issue: [Course Content](https://byui-cse.github.io/wdd130-ww-course/week02/prove-home-enhancement-css.html#:~:text=Change%20the%20existing%20paragraph%20p%20padding%20property%20value%20to%200%20pixels.%0ANote%20that%20any%200%20value%20does%20not%20need%20to%20specify%20the%20units).